packages/Toaster/src/toaster.scss

```diff
@@ -28,6 +28,68 @@
   --ids-toaster-z-index: #{tokens.$z-index-max};
```

Comment:
You can remove `--ids-toaster-overlay-padding` variable at the same time, the valkue is never used, only updated arround line 189.